src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -149,22 +156,30 @@ public void setVelocity(double velocity) {
             double volts = controller.calculate(getVelocity(), velocity);
             mainMotor.setVoltage(volts + feedforward.calculate(0, 0));
         } else {
-            mainMotor.set(ControlMode.Velocity, unitModel.toTicks100ms(velocity));
+            mainMotor.set(ControlMode.Velocity, unitModelPosition.toTicks100ms(velocity));
         }
     }
 
+
+    public void setAngleZero() {
+        double angle = unitModelGearbox.toUnits(mainMotor.getSelectedSensorPosition(1)
+                - Constants.Climber.ZERO_POSITION);
```

Comment:
`ZERO_POSITION = 0` so this does nothing. Should it just be `setPosition(onstants.Climber.ZERO_POSITION)`?

Reply:
No, the zero position changes according to the position of the absolute encoder, which we can't know right now without the robot...

Reply:
This function gets the absolute angle of the motor (relative to the horizon), and then we move the motor (using the internal encoder)